.github/workflows/main.yml

```diff
@@ -32,21 +32,12 @@ jobs:
     name: Run tests
     uses: ./.github/workflows/run-tests.yml
 
-  sonarcloud:
```

Comment:
Hi @HowardBraham, why it was removed from the main workflow? we do want to analyze `develop` branch